lesson3/code-tasks/task-4.js

```diff
@@ -0,0 +1,27 @@
+const path = '../users.json';
+
+const loadUsers = (path) => {
+  return fetch(path)
+    .then(response => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok.');
+      }
+
+      return response.json();
+    })
+    .then(users => {
+      let originalUsers = users
```

Comment:
No actual point of making an assignment to `originalUses`, since you're not using this value further in a program.